packages/shared/src/components/filters/AdvancedSettings.tsx

```diff
@@ -1,58 +1,23 @@
-import React, { ReactElement, useCallback, useMemo } from 'react';
+import React, { ReactElement } from 'react';
 import useFeedSettings from '../../hooks/useFeedSettings';
 import { FilterSwitch } from './FilterSwitch';
-import { AdvancedSettingsGroup } from '../../graphql/feedSettings';
 import { useAdvancedSettings } from '../../hooks/feed';
-import useTagAndSource from '../../hooks/useTagAndSource';
-import { Origin } from '../../lib/log';
-import { Source } from '../../graphql/sources';
 
 const ADVANCED_SETTINGS_KEY = 'advancedSettings';
 
 function AdvancedSettingsFilter(): ReactElement {
-  const { advancedSettings, isLoading, feedSettings } = useFeedSettings();
```

Comment:
All code here was moved inside the useFeedSettings hook and useAdvancedSettings without being changed so it could be easily reused in content type onboarding

Reply:
I like this export more, why don't we use this in the `ContentTypes` this makes more sense to me actually.9

Reply:
This is still exported and can be used for sure, just wanted to make sure to have the same behaviour so thought of exporting from here.
But makes sense, this is a bit more clear, just for not having duplicate code, thinking maybe we can do the filterings and what is needed in any common function

Reply:
Got back to this export for useFeedSettings